src/actions/CreateAction.php

```diff
@@ -41,6 +41,8 @@ public function run()
         $model->import_model = $this->controller->importModel;
         $model->setScenario(Import::SCENARIO_INSERT);
 
+        $model->user_id = $this->controller->user_id;
```

Comment:
```suggestion
        $model->user_id = \Yii::$app->getUser()->identity;
```
The package can also do this association

Reply:
@NgocTuanTruong can you make this change? If we do this, the change to `modules/admin/controllers/ImportNetworkTariffController.php` in https://github.com/webtoolsnz/samurai/pull/3356/files would no longer be required